superset/config.py

```diff
@@ -443,6 +443,7 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
     "ALLOW_FULL_CSV_EXPORT": False,
     "UX_BETA": False,
     "GENERIC_CHART_AXES": False,
+    "ALLOW_ADHOC_SUBQUERY": True,
```

Comment:
This should probably default to False since subqueries currently can violate access rules. We can change the default to True once we have more protections in place.

Reply:
I agree, even though this is technically a breaking change it prevents people from bypassing security rules.